bindings/client/src/error.rs

```diff
@@ -9,10 +10,11 @@ use std::io::ErrorKind;
 pub fn handle_api_err(err: ConductorApiError) -> anyhow::Error {
     match err {
         // Handle websocket closed errors by shutting down the process, as this is a fatal error
+        // for this agent.
         ConductorApiError::WebsocketError(e)
             if e.kind() == ErrorKind::Other && e.to_string() == "ConnectionClosed" =>
         {
-            panic!("Conductor API connection closed unexpectedly: {:?}", e)
+            AgentBailError::default().into()
```

Comment:
Normally:
- an error in the agent setup immediately stops that agent
- all errors are just logged during agent behaviour so that it can keep running
- errors in teardown are just logged because this is just a best effort cleanup

I'm happy to rename this error, but its purpose is to allow the agent to break out of its behaviour loop. If a HP goes down during a test we want to exit the loop, not keep spamming errors and wasting resources on the control node.

Reply:
I think that's ideal and was jokingly asking about some secret agent whose name must be Bail.